lambda/recipes-reindex/src/writeBatchToReindexQueue/config.ts

```diff
@@ -0,0 +1,7 @@
+import { mandatoryParameter } from 'lib/recipes-data/src/lib/parameters';
+import { RecipeIndexSnapshotBucket } from '../sharedConfig';
+
+export const getConfig = () => ({
+	RecipeIndexSnapshotBucket,
+	ReindexBatchSize: parseInt(mandatoryParameter('REINDEX_BATCH_SIZE')),
```

Comment:
Do we guard against this returning e.g. `NaN`?

Reply:
We don't do this [anyplace else in config](https://github.com/guardian/recipes-backend/blob/11aba448643d99974fee671868f4b664f2b54798/lib/recipes-data/src/lib/config.ts#L21) AFAICS, but we should probably fail fast in these cases! Another PR to address this more thoroughly across all apps? cc. @JustinPinner, as it relates to the comment above.